internal/pkg/manifest/worker_svc.go

```diff
@@ -206,6 +208,14 @@ func NewWorkerService(props WorkerServiceProps) *WorkerService {
 		svc.WorkerServiceConfig.TaskConfig.CPU = aws.Int(MinWindowsTaskCPU)
 		svc.WorkerServiceConfig.TaskConfig.Memory = aws.Int(MinWindowsTaskMemory)
 	}
+	if len(props.Topics) > 0 {
+		for idx, topic := range props.Topics {
+			if strings.Contains(aws.StringValue(topic.Name), ".fifo") {
+				props.Topics[idx].Name = aws.String(strings.TrimSuffix(aws.StringValue(topic.Name), ".fifo"))
+				svc.WorkerServiceConfig.Subscribe.Queue.FIFO.Enable = aws.Bool(true)
+			}
+		}
+	}
```

Comment:
It seems like but this for-loop we are saying as long as there is at least one fifo topic that the service subscribe to - even if the service could subscribe also to some standard topics - we create the default queue as fifo queue. Is this what we want?

Reply:
Yes, so when customer selects subscriptions from the wizard while initiating the worker service - 
1. There is no option for them to specify topic level sqs queue.
2. Only default sqs queue used to be created after selecting subscriptions.
3. Now that FIFO has been introduced, if customer selects FIFO topics then we will create default FIFO sqs queue as we can not create both default Standard and FIFO queues at the same time. 

So customers has to select either Standard or FIFO topic subscriptions. And we can convey the same through some warning may be.

Reply:
Would it be a bad idea if we populate the manifest such that:
1. If all subscribed topics are fifo, then we write `queue.fifo: true`
2. If all subscribed topics are standard, then we don't write `queue.fifo: true` (create default standard queue)
3. If some are fifo, then we keep the default queue as "standard, but populate the manifest so that each FIFO topics has their topic-sepecifc queue that is fifo.

Reply:
If the user selects FIFO and standard topics, why can't we generate a manifest that has standard queues for the standard topics?


Something like this in pseudo-code:
```
standardTopics <- filter(props.Topics, isStandard)
fifoTopics <- filter(props.Topics, isFifo)

if len(fifoTopics) == 0:
   return # nothing to do

enableEventsQueueWithFifo()
for standardTopic in standardTopics:
   enableStandardQueue(standardTopic)
```

Reply:
I think option 1 and 2 are valid. But 3rd doesn't seem consistent because we do not create topic-specific queues for standard ever reason being we create default standard queue. Moreover customer is now aquatinted with the behavior that when you subscribe topics through wizard are having only one default queue and not topic-specific ones.

Reply:
> If the user selects FIFO and standard topics, why can't we generate a manifest that has standard queues for the standard topics?
> 
> Something like this in pseudo-code:
> 
> ```
> standardTopics <- filter(props.Topics, isStandard)
> fifoTopics <- filter(props.Topics, isFifo)
> 
> if len(fifoTopics) == 0:
>    return # nothing to do
> 
> enableEventsQueueWithFifo()
> for standardTopic in standardTopics:
>    enableStandardQueue(standardTopic)
> ```

so you mean `enableEventsQueueWithFifo()` will create fifo queue to which all the fifo topics will be subscribed to?

Reply:
To clarify, I meant:
1. The manifest that we generate would be:
```
subscribe:
  topics:
     - name: myFIFOTopicA
     - name: myFIFOTopicB
   queue:
      fifo: true
```
2. The manifest generated would be
```
subscribe:
  topics:
     - name: myStandardTopicA
     - name: myStandardTopicB
   queue:
```
3. The manifest would be
```
subscribe:
  topics:
     - name: myFIFOTopicA
        queue:
            fifo: true
     - name: myStandardTopicA
   queue:
```

Reply:
I understood your point, but I am just hesitant because of inconsistency between standard and FIFO implementation. 
What is customers want FIFO to be the default queue and standard being the topic-specific queue. That will still be impossible through the CLI.

Reply:
People can still modify the manifest after `svc init`. 

The purpose of generating a default manifest by `svc init` is to give users a good start, not to produce a manifest that they are stuck with forever - so I think it's okay for us to make a decision between: A. create a default standard + topic-specific fifo; B. default fifo + topic-specific standard. 

If we generate a manifest that goes with A, and the user wants b, they can modify the manfiest; and vice versa.